desktopApp/reactApp/src/pages/ConsortiumDetails/ConsortiumDetailsPage.tsx

```diff
@@ -8,13 +7,32 @@ import { useUserState } from "../../contexts/UserStateContext";
 import StartRunButton from "./StartRunButton/StartRunButton";
 import { ConsortiumDetailsProvider } from "./ConsortiumDetailsContext";
 import { LatestRun } from "./LatestRun/LatestRun";
-import ComputationDisplay from "./StudyConfiguration/Computation/ComputationDisplay";
+import ComputationDisplay from "./ComputationDisplay/ComputationDisplay";
 import ConsortiumLeaderNotes from "./ConsortiumLeaderNotes/ConsortiumLeaderNotes";
+import Computation from "./Computation/Computation";
+import ComputationParameters from "./ComputationParameters/ComputationParameters";
+import { useNavigate } from 'react-router-dom';
+import { useCentralApi } from "../../apis/centralApi/centralApi";
 
 export default function ConsortiumDetailsPage() {
     const { userId } = useUserState();
     const { data, status, refetch, isLeader } = useConsortiumDetails();
-    const { studyConfiguration, members, activeMembers, readyMembers, leader, title, description } = data;
+    const { consortiumId, studyConfiguration, members, activeMembers, readyMembers, leader, title, description } = data;
+    const navigate = useNavigate();
+
+    const { consortiumLeave } = useCentralApi();
+
+    // Handle leaving the consortium
+    const handleLeave = async () => {
```

Comment:
handleLeave is never called. This makes it unnecessary to extract consortiumId from data and it makes it unnecessary to import `useCentralApi`

Reply:
I missed removing that when I place the leave button inside the members panel. I'll remove.